sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -180,7 +180,7 @@ def get_queue_receiver(self, queue_name, **kwargs):
          will be immediately removed from the queue, and cannot be subsequently rejected or re-received if
          the client fails to process the message. The default mode is PeekLock.
         :paramtype mode: ~azure.servicebus.ReceiveSettleMode
-        :keyword float idle_timeout: The timeout in seconds between received messages after which the receiver will
+        :keyword float max_wait_time: The timeout in seconds between received messages after which the receiver will
```

Comment:
we need to remove "after which the receiver will automatically shutdown" as it's not closed now.